src/Components/NavBar/index.js

```diff
@@ -0,0 +1,40 @@
+import React from "react";
+
+const Navbar = () => {
```

Comment:
Remove the additional javascript file, just leave one

Reply:
Done